doc/Makefile

```diff
@@ -4,7 +4,8 @@
 
 BUILDDIR ?= _build
 DOC_TAG ?= development
-SPHINXOPTS ?= -j auto -W --keep-going -T
+SPHINX_PROCESSES ?= auto
```

Comment:
nit: `SPHINX_JOBS` (`JOBS` would probably be fine too IMO)

Reply:
Thank you for the suggestion, and I applied it.